### ISSUE TITLE
Selection not always preserved on imageUrl/resource changes

### ISSUE THREAD

**Issue by fredl11** [melistik/vaadin-jcrop#1]
The active selection is not always preserved when setImageUrl() or setResource() is called. I can reproduce the issue with the [sampler](http://non-rocket-science.jelastic.servint.net/vaadin-jcrop/):
1. Make a random selection
2. Click "switchImage" or "setResource"

Result:
- When the image is already cached, switchImage works for me,
- setResource seems to loose the selection every time.

The information about the selection above the image however shows the correct values. Only the overlay is lost.

It seems as if the problem is related to the image loading speed. If the the image is served slowly, the selection overlay is lost.

**Comment by melistik** [comment]
yes you are right... i will check it out and try to fix it for the next release - but it take some time...
